packages/skill-lesson/video/video-overlays.tsx

```diff
@@ -403,7 +403,7 @@ const BuyProduct: React.FC<{product?: SanityProduct}> = ({product}) => {
         <h3 data-description="">
           <Balancer>
             {canViewRegionRestriction ? (
-              <div data-markdown="">
```

Comment:
wouldn't it be possible to use the markdown data attribute instead of adding a new one? this addition would need to be styled across all products.

Reply:
evntually removed `data-markdown` at all, no additional styles needed.